applications/vessel-history/src/service-worker.ts

```diff
@@ -64,6 +64,9 @@ self.addEventListener('message', (event) => {
 
 // Any other custom service worker logic can go here.
 
+// Precache GeoJson for offline use
+precacheAndRoute([{ url: '/data/ne_10m_admin_0_countries_gj.geojson' }])
```

Comment:
If feasible I'd share the path here and within the generator to ensure if the file changes it is updated in both sides

Reply:
Done